chapters/qa-01.mdx

```diff
@@ -70,7 +70,7 @@ Možná struktura je tato:
 
 -   Pages, kde můžeš mít další html stránky
 -   Design, kde můžeš mít CSS
--   Images, kde můžeš mít obrázky. V našem případě jde o složku obrazky. Jsme krásně česko anglicky nekonzistentní, tomu se doporučujeme vyhnout.
+-   Images, kde můžeš mít obrázky. V našem případě jde o složku obrázky. Jsme krásně česko anglicky nekonzistentní, tomu se doporučujeme vyhnout.
```

Comment:
Jméno složky je skutečně `obrazky`, nikoliv `obrázky`. Možná nějak zvýraznit, ať je jasné, že nejde o špatné slovo?

Reply:
přidat "s názvem"

Reply:
lepší dát slovo obrazky do uvozovek jakožto literal "obrazky"